src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -1059,6 +1068,34 @@ public static FormulaValue OptionSetValueToLogicalName(IRContext irContext, Opti
             var optionSet = args[0];
             var logicalName = optionSet.Option;
             return new StringValue(irContext, logicalName);
+        }
+
+        public static FormulaValue PlainText(IRContext irContext, StringValue[] args)
```

Comment:
>public static FormulaValue PlainText(IRContext irContext, StringValue[] args) [](http://example.com/codeflow?start=8&length=77)

How `PlainText` function should handle a blank value? PA returns `Blank()` while this implementation is returning `Error({Kind:ErrorKind.InvalidArgument})`

Reply:
Checked on PowerApps side and updated logic to allow formularvalue input to fix here. Thanks.

Reply:
updated